### PR TITLE
Added receive argument to entrypoint

### LINKED ISSUE #75: [bug] GitHub Action hanging
The github action doesn't appear to work. I've cancelled a couple of tasks after 15min of activity and the logs show the following after getting everything setup

```
...
Already have image (with digest): gcr.io/github-actions-images/action-runner:latest
14:35:13.811Z  INFO probot: Listening on http://localhost:3000
14:35:13.814Z  INFO probot:
  
  Welcome to Probot! Go to http://localhost:3000 to get started.
```

### PR DESCRIPTION
**Pull request by SvanBoxel**
Added receive argument to docker entrypoint per probot documentation. https://probot.github.io/docs/deployment/#github-actions

Hopefully fixes #75.

### CODE REVIEW

**Comment by SvanBoxel** [comment]
@matthewnessworthy @rtimush @mkj28  Could you all give this branch a try with github actions?

**Comment by mkj28** [comment]
@SvanBoxel for me fails with:
```
### STARTED SvanBoxel/delete-merged-branch@37738da38f5ddfe2187d0cb8a884effff74a09ce 16:33:53Z

Pulling image: gcr.io/github-actions-images/action-runner:latest
latest: Pulling from github-actions-images/action-runner
169185f82c45: Pulling fs layer
0ccde4b6b241: Pulling fs layer
d0372f57daa2: Pulling fs layer
165911d108d6: Pulling fs layer
54996bce1de5: Pulling fs layer
165911d108d6: Waiting
54996bce1de5: Waiting
d0372f57daa2: Verifying Checksum
d0372f57daa2: Download complete
169185f82c45: Verifying Checksum
169185f82c45: Download complete
0ccde4b6b241: Verifying Checksum
0ccde4b6b241: Download complete
54996bce1de5: Verifying Checksum
54996bce1de5: Download complete
165911d108d6: Verifying Checksum
165911d108d6: Download complete
169185f82c45: Pull complete
0ccde4b6b241: Pull complete
d0372f57daa2: Pull complete
165911d108d6: Pull complete
54996bce1de5: Pull complete
Digest: sha256:c9bb432ec5ec08ee08b040a9fccacebbbf8a91444dac4721600cf5dca9dae57e
Status: Downloaded newer image for gcr.io/github-actions-images/action-runner:latest
Step 1/15 : FROM node:10.11.0-alpine
10.11.0-alpine: Pulling from library/node
4fe2ade4980c: Pulling fs layer
0b82511f7bf4: Pulling fs layer
e2bae3466659: Pulling fs layer
e2bae3466659: Verifying Checksum
e2bae3466659: Download complete
4fe2ade4980c: Verifying Checksum
4fe2ade4980c: Download complete
0b82511f7bf4: Verifying Checksum
0b82511f7bf4: Download complete
4fe2ade4980c: Pull complete
0b82511f7bf4: Pull complete
e2bae3466659: Pull complete
Digest: sha256:fcd9b3cb2fb21157899bbdb35d1cdf3d6acffcd91ad48c1af5cb62c22d2d05b1
Status: Downloaded newer image for node:10.11.0-alpine
 ---> 5206c0dd451a
Step 2/15 : LABEL "com.github.actions.name"="Delete merged branch"
 ---> Running in e6b95a7e8d41
 ---> a291f98dbf34
Step 3/15 : LABEL "com.github.actions.description"="No more manually deleting merged branches, this lovely app does it for you."
 ---> Running in 15ac8351498c
 ---> 5d2aa7f3b511
Step 4/15 : LABEL "com.github.actions.icon"="archive"
 ---> Running in 99e0d7626cf4
 ---> c41106df9004
Step 5/15 : LABEL "com.github.actions.color"="red"
 ---> Running in cd561c93e810
 ---> 1a509ce7c682
Step 6/15 : LABEL "repository"="https://github.com/SvanBoxel/delete-merged-branch"
 ---> Running in 94b97574fb45
 ---> d021a1eaa442
Step 7/15 : LABEL "homepage"="https://github.com/SvanBoxel"
 ---> Running in a19c9a8f3a9d
 ---> a0e3b13ce428
Step 8/15 : LABEL "maintainer"="svboxel@gmail.com"
 ---> Running in 39418c115f1c
 ---> 1db8df56c9e0
Step 9/15 : ENV PATH=$PATH:/app/node_modules/.bin
 ---> Running in 1e8ba94a86f2
 ---> 0b3da51fdf41
Step 10/15 : WORKDIR /app
 ---> Running in b995d9bb8726
 ---> 10d9db1b3cdd
Step 11/15 : COPY package.json yarn.lock /app/
 ---> ce59a336a211
Step 12/15 : RUN yarn install --production
 ---> Running in d93eb6b062f7
yarn install v1.9.4
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
(node:1) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
info fsevents@1.2.4: The platform "linux" is incompatible with this module.
info "fsevents@1.2.4" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@1.2.7: The platform "linux" is incompatible with this module.
info "fsevents@1.2.7" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
[5/5] Building fresh packages...
Done in 26.63s.
 ---> 3b00589f31e4
Step 13/15 : COPY . .
 ---> ae1824117763
Step 14/15 : ENTRYPOINT ["probot", "receive"]
 ---> Running in d5103c7cbeb9
 ---> 3d4aa72ad340
Step 15/15 : CMD ["run", "/app/index.js"]
 ---> Running in de55c8331f4c
 ---> 2686f683e213
Successfully built 2686f683e213
Successfully tagged gcr.io/gct-12-v39xk78y8sw7hj4y6hz-9bs/93f9d22e967174b5c3fead73f900549fddaef899f75e094e84de5f33cf3f98d1/8a5edab282632443219e051e4ade2d1d5bbc671c781051bf1437897cbdfea0f1:c42d4806401068abd1a3d4062cceecce1f5752325b26f2504fc470e6becaee7c
Already have image (with digest): gcr.io/github-actions-images/action-runner:latest
/app/node_modules/resolve/lib/sync.js:45
    throw err;
    ^

Error: Cannot find module 'run' from '/github/workspace'
    at module.exports (/app/node_modules/resolve/lib/sync.js:43:15)
    at Object.exports.resolve (/app/node_modules/probot/lib/resolver.js:10:20)
    at Probot.load (/app/node_modules/probot/lib/index.js:203:32)
    at /app/node_modules/probot/lib/index.js:224:78
    at Array.forEach (<anonymous>)
    at Probot.setup (/app/node_modules/probot/lib/index.js:224:38)
    at Object.<anonymous> (/app/node_modules/probot/bin/probot-receive.js:42:8)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
    at startup (internal/bootstrap/node.js:279:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:752:3)

### FAILED SvanBoxel/delete-merged-branch@37738da38f5ddfe2187d0cb8a884effff74a09ce 16:35:03Z (1m9.748s)
```

**Comment by mkj28** [comment]
here's my .github/main.workflow:
```
workflow "delete merged branch" {
  on = "pull_request"
  resolves = ["SvanBoxel/delete-merged-branch@37738da38f5ddfe2187d0cb8a884effff74a09ce"]
}

action "SvanBoxel/delete-merged-branch@37738da38f5ddfe2187d0cb8a884effff74a09ce" {
  uses = "SvanBoxel/delete-merged-branch@37738da38f5ddfe2187d0cb8a884effff74a09ce"
  secrets = ["GITHUB_TOKEN"]
}
```

File: Dockerfile [hunk 1]
```diff
@@ -19,5 +19,5 @@ COPY package.json yarn.lock /app/
 RUN yarn install --production
 COPY . .
 
-ENTRYPOINT ["probot"]
+ENTRYPOINT ["probot", "receive"]
 CMD ["run", "/app/index.js"]
```
Comment (mkj28) [inline Dockerfile]: @SvanBoxel 
```suggestion
CMD ["/app/index.js"]
```

Comment (mkj28) [reply Dockerfile]: per https://probot.github.io/docs/deployment/#github-actions

Comment (mkj28) [reply Dockerfile]: I can confirm it works if you change CMD to this

**Comment by SvanBoxel** [comment]
Accepted your suggestion, seems to work indeed. Thanks!

**Pull request merged**